oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -250,4 +255,13 @@ private static long getFileDescriptors(int index) {
         }
         return 0L;
     }
+
+    private static long getFileDescriptorsPerProcess() {
+        final String filename = ProcPath.SYS_FS_FILE_MAX;
+        final List<String> fileContent = FileUtil.readFile(filename);
+        if (!fileContent.isEmpty()) {
+            return ParseUtil.parseLongOrDefault(fileContent.get(0), 0L);
```

Comment:
We have a `getLongFromFile()` method on the fileUtil class that does exactly this logic.